### ISSUE TITLE
[Enhancement]: Replace `ImageKit` component with `<img>`

### ISSUE THREAD

**Issue by leonardoarcari** [leonardoarcari/dcapal#54]
# **🚀 Enhancement Request**

Now that we introduced Webpack's Asset Modules, there's no longer need for `ImageKit` component: it's cumbersome, makes image handling hard and it doesn't speed up image fetching at all. DcaPal is served behind Cloudflare which already does all the caching we need.

## **Describe the solution you'd like**

Let's replace `ImageKit` with good old `<img>` tags importing assets from `/images` folder

**Comment by corradopetrelli** [comment]
Some steps as reminder:

- [ ] Save all images on images folder (preferably with the same structure)
- [ ] On `webpack.common.js` integrate the field alias like that:
```
 resolve: {
   ...
   alias: {
      "@images": path.resolve(__dirname, 'images/'),
   }
 }
```
- [ ] Replace all image that use `<IKImage>` with `<img>`  with the folder imported like that:
```
import fooImage from '@images/fooFolder/foo.svg'
...
<img src={githubImage} alt="Foo"/>
```
- [ ] Remove `dcapal-frontend/src/app/images.js`
- [ ] Remove `imagekitio-react` package and all reference of Imagekit urls (check also README.md)

**Comment by leonardoarcari** [comment]
> ```js
> alias: {
>      "@images": path.resolve(__dirname, 'images/'),
>   }
>```

Oh nice trick, wasn't aware of that 🙂 Is it portable to Vite too?

**Comment by corradopetrelli** [comment]
> Is it portable to Vite too?

Yes, of course!

**Comment by leonardoarcari** [comment]
Resolved in #57
